packages/eslint-config/package.json

```diff
@@ -2,65 +2,62 @@
   "name": "@10up/eslint-config",
   "version": "2.4.6",
   "description": "A shareable ESLint configuration",
-	"publishConfig": {
-    "access": "public"
+  "publishConfig": {
+	"access": "public"
   },
   "scripts": {
-    "lint": "eslint .",
-    "test": "npm run test:index && npm run test:react && npm run test:wordpress && npm run test:node",
-    "test:index": "node fixtures/test-eslint-config --config-file=./index.js",
-    "test:react": "node fixtures/test-eslint-config --config-file=./react.js",
-    "test:wordpress": "node fixtures/test-eslint-config --config-file=./wordpress.js",
-    "test:node": "node fixtures/test-eslint-config --config-file=./node.js"
+	"lint": "eslint .",
+	"test": "node fixtures/test-eslint-config.js",
+	"test:index": "node fixtures/test-eslint-config.js --config-file=index",
+	"test:react": "node fixtures/test-eslint-config.js --config-file=react",
+	"test:wordpress": "node fixtures/test-eslint-config.js --config-file=wordpress",
+	"test:node": "node fixtures/test-eslint-config.js --config-file=nodejs"
+  },
+  "exports": {
```

Comment:
Would this require a major bump?

Reply:
No.

This was a major concern. I didn't want to create a breaking change. Renaming node to nodejs would have been breaking so I tried with exports to see if it worked. 

The exports change just required changes on some internal routes but no changes on external package usages.